ports/xlsxio/vcpkg.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "xlsxio",
+  "version-string": "2021-03-24+0100",
+  "description": "Cross-platform C library for reading values from and writing values to .xlsx files",
+  "homepage": "https://github.com/brechtsanders/xlsxio",
+  "dependencies": [
+    "expat",
+    {
+      "name": "minizip",
+      "features": [
+        "bzip2"
+      ]
+    }
+  ],
+  "features": {
+    "libzip": {
+      "description": "use libzip instead of minizip",
+      "dependencies": [
+        {
+          "name": "libzip",
+          "features": [
+            "bzip2"
+          ]
+        }
+      ]
+    }
+  }
```

```suggestion
  ]
```

Features cannot be used to model replacements like "Use X instead of Y" -- features must only be additive.

In this case, we must choose the most reasonable default (minizip seems fine) and users that want the other configuration can use overlay ports.

---

It's ok, the author also suggests `minizip`, and `libzip` causes a bug that libreoffice can't open xlsx files that edited when using `libzip`.